### ISSUE TITLE
Marvel's Superhero Predicting Game

### ISSUE THREAD

**Issue by Manasi2001** [prathimacode-hub/ML-ProjectKart#152]
Hey LGM-SOC participant, great to have you here! 😀
Use this interesting [dataset](https://www.kaggle.com/dannielr/marvel-superheroes) and try to build a game that is able to recognize the Marvel Superhero based on his/her various characteristics.
Happy coding :)

**Comment by vaibhavgupta2499** [comment]
Can you assign me this issue as LGMSOC21 participant ?

**Comment by prathimacode-hub** [comment]
Yes, let me know of your approach you would follow, algos you would go with and task you are choosing up. So that I could assign it accordingly. @vaibhavgupta2499

**Comment by vaibhavgupta2499** [comment]
I am aiming to use multiple category classification. Will that suffice ? Also there are multiple datasets, so should I work on all of them ? My approach will be to first make each character (name) as a separate column, then input True/ False if that particular entry belongs to that character or not.

**Comment by prathimacode-hub** [comment]
It's upto your choice to choose single or mutiple dataset, yeah multiple category classification would be better as we can see the efficiency and compare the models. That's great. You can proceed. Issue assigned. @vaibhavgupta2499

**Comment by prathimacode-hub** [comment]
No update from you yet, It's been 12days since issue is assigned. @vaibhavgupta2499

**Comment by vaibhavgupta2499** [comment]
@prathimacode-hub Mam, Please just give me one more day, I will surely complete the project by tomorrow.

**Comment by prathimacode-hub** [comment]
Ok. @vaibhavgupta2499

**Comment by vaibhavgupta2499** [comment]
Mam, I have been able to make models for prediction, but the results are not so good. I have user two models, one is Decision Tree, and other is Logistic Regression. As there is only one instance of each class. Please see this file - https://github.com/vaibhavgupta2499/ColabWorks/blob/main/marvel_char.ipynb

**Comment by prathimacode-hub** [comment]
Is everything going fne? @vaibhavgupta2499

**Comment by vaibhavgupta2499** [comment]
Yes Mam, working on the problems. Will update you with the details in a day or two. Thank you mam.

**Comment by vaibhavgupta2499** [comment]
Mam, I have been working on the project, implemented different models, but the accuracy is Zero. Can you please help. The code is here - https://github.com/vaibhavgupta2499/ColabWorks/blob/main/marvel_char.ipynb

**Comment by prathimacode-hub** [comment]
It's because classes are not defined properly. Seems to be change in type of classes. @vaibhavgupta2499

**Comment by prathimacode-hub** [comment]
Are you able to do it or not? As last date to merge PR's is tomorrow. If not done by tomorrow, I would have to remove the label. @vaibhavgupta2499

**Comment by vaibhavgupta2499** [comment]
Please give me time till tomorrow. I will try my best to complete it by then.

**Comment by prathimacode-hub** [comment]
Ok, but I need to submit the report by evening. Make sure you do it before 4pm. @vaibhavgupta2499

**Comment by vaibhavgupta2499** [comment]
Please un-assign me this. Can I take up some other issue ?

**Comment by prathimacode-hub** [comment]
Sorry to say, But time is up now. Team has winded it up. They did say no more PR's with respect to the program. No worries,  I can assign you other issue but it won't be counted under LGMSOC program. If u r fine with it, I shall do. @vaibhavgupta2499

**Comment by vaibhavgupta2499** [comment]
@prathimacode-hub No thank you. I will contribute to this repo, later, independently. It was nice working with you.
